lib/getConfig.js

```diff
@@ -24,8 +29,8 @@ var inquirer = require('inquirer'),
     }, {
         type: 'input',
         name: 'updateCmd',
-        message: 'Shell command to be issued to fetch the new version of the app. Usually git pull origin master',
-        default: 'git pull origin master'
+        message: 'Shell command to be issued to fetch the new version of the app. `{branch}` will replace by branch name.',
```

Comment:
The English here is a little weird. Should this say `'{branch}' will be replaced by branch name.`?

Reply:
Thank you pointed out. My english is not good.

Reply:
I fixed the points that were pointed out.